test/gtest/uct/ib/test_cqe_zipping.cc

```diff
@@ -41,7 +41,7 @@ class test_cqe_zipping : public test_uct_ib_with_specific_port {
          * Local deadline prevents hanging inside receive loop in case of
          * connectivity issues
          */
-        const ucs_time_t deadline = ucs::get_deadline(60);
+        const ucs_time_t deadline = ucs::get_deadline(600);
```

Comment:
comment that need longer time to wait for zip to happen

Reply:
unhandled